awstesting/client.go

```diff
@@ -5,6 +5,7 @@ import (
 	"github.com/aws/aws-sdk-go/aws/client"
 	"github.com/aws/aws-sdk-go/aws/client/metadata"
 	"github.com/aws/aws-sdk-go/aws/defaults"
+	"github.com/aws/aws-sdk-go/awstesting/mock"
```

Comment:
`aws/client`shouldn't import a testing library  This will pull the testing dependancies into runtime of the SDK